src/Http/ApiClient.php

```diff
@@ -194,8 +194,8 @@ public function put($endpoint, array $data)
                 'headers' => ['Content-Type' => 'application/json; charset=utf-8'],
                 'json' => $data
             ]);
-
-            return $response;
+            
```

Comment:
Be careful not to commit white space. You can configure most code editors to delete trailing whitespace on save.